### PR TITLE
Sven

### PR DESCRIPTION
**Pull request by beniwein**

### CODE REVIEW

**Review by amstutzsven — COMMENTED** [review]
testtesttest

File: webpack.mix.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ const mix = require('laravel-mix');
  |
  */
 
-mix.js('resources/js/app.js', 'public/js').postCss('resources/css/app.css', 'public/css', [
-    require('tailwindcss'),
-    require('autoprefixer'),
-]);
+ mix.js('resources/js/app.js', 'public/js')
```
Comment (amstutzsven) [inline webpack.mix.js:14]: aölkfjösa

File: package-lock.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "html",
+    "name": "big-data",
     "lockfileVersion": 2,
     "requires": true,
```
Comment (amstutzsven) [inline package-lock.json:4]: awefd

File: package-lock.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
                 "autoprefixer": "^10.4.2",
                 "axios": "^0.25",
                 "bootstrap": "^5.1.3",
-                "laravel-mix": "^6.0.6",
+                "laravel-mix": "^6.0.49",
                 "lodash": "^4.17.19",
                 "postcss": "^8.4.6",
```
Comment (amstutzsven) [inline package-lock.json:19]: sdf

File: package.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
         "autoprefixer": "^10.4.2",
         "axios": "^0.25",
         "bootstrap": "^5.1.3",
-        "laravel-mix": "^6.0.6",
+        "laravel-mix": "^6.0.49",
```
Comment (amstutzsven) [inline package.json:19]: s<d

File: public/css/app.css [hunk 5]
```diff
@@ -1,3 +1,10262 @@
+@import url(https://fonts.googleapis.com/css?family=Nunito);
+@charset "UTF-8";
+/*!
+ * Bootstrap v5.1.3 (https://getbootstrap.com/)
+ * Copyright 2011-2021 The Bootstrap Authors
+ * Copyright 2011-2021 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
```
Comment (amstutzsven) [inline public/css/app.css:7]: asdfycsafdd

File: public/css/app.css [hunk 6]
```diff
@@ -355,7 +10614,7 @@ iframe,
 embed,
 object {
   display: block; /* 1 */
```
Comment (amstutzsven) [inline public/css/app.css:10616]: <ycx<c

File: public/css/app.css [hunk 7]
```diff
@@ -355,7 +10614,7 @@ iframe,
...
-  /*vertical-align: middle; /* 2 */
+  vertical-align: middle; /* 2 */
```
Comment (amstutzsven) [inline public/css/app.css:10617]: <sdf

File: public/css/app.css [hunk 8]
```diff
@@ -682,14 +10941,14 @@ select {
 .relative {
   position: relative;
 }
-.top-0 {
-  top: 0px;
+.left-0 {
+  left: 0px;
```
Comment (amstutzsven) [inline public/css/app.css:10945]: ds

File: resources/sass/app.scss [hunk 9]
```diff
@@ -6,3 +6,13 @@
 
 // Bootstrap
 @import '~bootstrap/scss/bootstrap';
+
+.body {$font-stack: Helvetica, sans-serif;
```
Comment (amstutzsven) [inline resources/sass/app.scss:10]: c<yvdsd

**Review by amstutzsven — COMMENTED** [review]
MIGRATION FIX

**Comment by beniwein** [comment]
Refresh to Main

**Pull request closed without merging**